sdk/src/main/java/com/silanis/esl/sdk/service/apiclient/AccountConfigClient.java

```diff
@@ -82,4 +82,65 @@ public void deleteHandoverUrl(String language) {
             throw new EslException("Could not delete handover url.", e);
         }
     }
+
+    public List<String> createDeclineReasons(String language, List<String> declineReasons) {
+        String path = template.urlFor(UrlTemplate.DECLINE_REASONS_URL_PATH)
+                .replace("{language}", language)
+                .build();
+        String json = Serialization.toJson(declineReasons);
+
+        try {
+            String stringResponse = restClient.post(path, json);
+            return Serialization.fromJsonToList(stringResponse, String.class);
+
+        } catch (RequestException e) {
+            throw new EslServerException("Could not create decline reasons.", e);
+        } catch (Exception e) {
+            throw new EslException("Could not create decline reasons.", e);
+        }
+    }
+
+    public List<String> updateDeclineReasons(String language, List<String> declineReasons) {
+        String path = template.urlFor(UrlTemplate.DECLINE_REASONS_URL_PATH)
+                .replace("{language}", language)
+                .build();
+        String json = Serialization.toJson(declineReasons);
+
+        try {
+            String stringResponse = restClient.put(path, json);
+            return Serialization.fromJsonToList(stringResponse, String.class);
+
+        } catch (RequestException e) {
+            throw new EslServerException("Could not create decline reasons.", e);
+        } catch (Exception e) {
+            throw new EslException("Could not create decline reasons.", e);
+        }
+    }
+
+    public List<String> getDeclineReasons(String language) {
+        String path = template.urlFor(UrlTemplate.DECLINE_REASONS_URL_PATH)
+                .replace("{language}", language)
+                .build();
+        try {
+            String stringResponse = restClient.get(path);
+            return Serialization.fromJsonToList(stringResponse, String.class);
+        } catch (RequestException e) {
+            throw new EslServerException("Could not create decline reasons.", e);
+        } catch (Exception e) {
+            throw new EslException("Could not create decline reasons.", e);
+        }
+    }
+
+    public void deleteDeclineReasons(String language) {
+        String path = template.urlFor(UrlTemplate.DECLINE_REASONS_URL_PATH)
+                .replace("{language}", language)
+                .build();
+        try {
+            restClient.delete(path);
+        } catch (RequestException e) {
+            throw new EslServerException("Could not create decline reasons.", e);
+        } catch (Exception e) {
+            throw new EslException("Could not create decline reasons.", e);
```

Comment:
The error message "Could not create decline reasons" is incorrect. Please change to "Could not delete decline reasons".